tools/benchmark/cmd/root.go

```diff
@@ -48,10 +46,3 @@ func init() {
 	RootCmd.PersistentFlags().UintVar(&totalConns, "conns", 1, "Total number of gRPC connections")
 	RootCmd.PersistentFlags().UintVar(&totalClients, "clients", 1, "Total number of gRPC clients")
 }
-
-func Execute() {
```

Comment:
> Change cmd.RootCmd.Execute() to cmd.Execute()?

Of course it will work but I just removing the Execute() will result a little bit fewer code and simpler.